setup.md

```diff
@@ -121,108 +121,35 @@ $ git config --global core.editor "nano -w"
 ~~~
 {: .language-bash}
 
-### GitHub Account
-GitHub is a free, online host for Git repositories that you will use during the course to store your code in so 
-you will need to open a free [GitHub](https://github.com/) account unless you do not already have one.
+### GitLab Account
+GitLab is an online host for Git repositories that you will use during the course to store your code in.
+You will need a GitLab account with which you have access to TNOs GitLab instance. 
```

Comment:
We lost the link here.
```suggest
You will need a [GitLab](https://gitlab.com/) account with which you have access to TNOs GitLab instance.
```

Reply:
Yes, un purpose. Because they will be used to a different link (i.e. something like tno.gitlab.com)